cmake/spore_codegen.cmake

```diff
@@ -2,7 +2,7 @@ function(spore_codegen SPORE_TARGET_NAME)
   cmake_parse_arguments(
     "SPORE_CODEGEN"
     "DEBUG;SEQUENTIAL;FORCE;REFORMAT"
-    "CONFIG;CACHE;CXX_STANDARD;TARGET_NAME;BIN_NAME"
+    "CONFIG;CACHE;CXX_STANDARD;TARGET_NAME;BIN_NAME;OUTPUT;PREFIX"
```

Comment:
Can we supply `WORKING_DIRECTORY` directly? I'd prefer to let the user have full control over the value instead of partial control.

Reply:
Sure, will do.